### PR TITLE
[Bot] Update docs for latest fastlane release (actions.md, available-plugins.md) 🚀

### PR DESCRIPTION
**Pull request by revolter**
Auto-generated by _fastlane_ bot 🤖

### CODE REVIEW

File: docs/actions/adb_devices.md [hunk 1]
```diff
@@ -39,11 +39,12 @@ end
...
 
-Key | Description
-----|------------
-  `adb_path` | The path to your `adb` binary
-
+Key | Description | Default
+----|-------------|--------
+  `adb_path` | The path to your `adb` binary | adb
 
+_- = this parameter doesn't have a default value_<br/>
```
Comment (janpio) [inline docs/actions/adb_devices.md]: This `<br/>` could probably be replaced with `  ` (double space)

File: docs/actions/adb_devices.md [hunk 2]
```diff
@@ -39,11 +39,12 @@ end
 
 ## Parameters
 
-Key | Description
-----|------------
-  `adb_path` | The path to your `adb` binary
-
+Key | Description | Default
+----|-------------|--------
+  `adb_path` | The path to your `adb` binary | adb
```
Comment (janpio) [inline docs/actions/adb_devices.md]: Maybe `adb` (as code instead of just "adb"?)

File: docs/actions/add_extra_platforms.md [hunk 3]
```diff
@@ -35,11 +35,12 @@ add_extra_platforms(
 
 ## Parameters
 
-Key | Description
-----|------------
-  `platforms` | The optional extra platforms to support
-
+Key | Description | Default
+----|-------------|--------
+  `platforms` | The optional extra platforms to support |
```
Comment (janpio) [inline docs/actions/add_extra_platforms.md]: What does no value mean? Shouldn't really happen, right?

File: docs/actions/add_git_tag.md [hunk 4]
```diff
@@ -53,18 +55,20 @@ add_git_tag(
...
-  `message` | The tag message. Defaults to the tag's name
-  `commit` | The commit or object where the tag will be set. Defaults to the current HEAD
-  `force` | Force adding the tag
-  `sign` | Make a GPG-signed tag, using the default e-mail address's key
-
-
+Key | Description | Default
+----|-------------|--------
+  `tag` | Define your own tag text. This will replace all other parameters | -
+  `grouping` | Is used to keep your tags organised under one 'folder'. Defaults to 'builds' | builds
```
Comment (janpio) [inline docs/actions/add_git_tag.md]: Test like "Defaults to 'builds'" should probably be removed then from the description, right?

File: docs/actions/backup_xcarchive.md [hunk 5]
```diff
@@ -38,15 +38,16 @@ backup_xcarchive(
...
-  `zip_filename` | Filename of the compressed archive. Will be appended by `.xcarchive.zip`. Default value is the output xcarchive filename
-  `versioned` | Create a versioned (date and app version) subfolder where to put the archive. Default value `true`
-
-
+Key | Description | Default
+----|-------------|--------
+  `xcarchive` | Path to your xcarchive file. Optional if you use the `xcodebuild` action | -
+  `destination` | Where your archive will be placed | -
+  `zip` | Enable compression of the archive. Default value `true` | true
+  `zip_filename` | Filename of the compressed archive. Will be appended by `.xcarchive.zip`. Default value is the output xcarchive filename | -
```
Comment (janpio) [inline docs/actions/backup_xcarchive.md]: What is going on here? Shouldn't there be something per the desc?

File: docs/actions/bundle_install.md [hunk 6]
```diff
@@ -25,28 +25,29 @@ Author | @birmacher, @koglinjg
...
+  `system` | Install to the system location ($BUNDLE_PATH or $GEM_HOME) even if the bundle was previously installed somewhere else for this application | false
+  `quiet` | Only output warnings and errors | false
+  `retry` | Retry network and git requests that have failed | -
+  `shebang` | Specify a different shebang executable name than the default (usually 'ruby') | -
+  `standalone` | Make a bundle that can work without the Bundler runtime | -
+  `trust_policy` | Sets level of security when dealing with signed gems. Accepts `LowSecurity`, `MediumSecurity` and `HighSecurity` as values | -
+  `without` | Exclude gems that are part of the specified named group | -
+  `with` | Include gems that are part of the specified named group | -
+
+_- = this parameter doesn't have a default value_<br/>
```
Comment (janpio) [inline docs/actions/bundle_install.md]: to get rid of the "this":

Comment (janpio) [reply docs/actions/bundle_install.md]: - = no default value for this parameter
* = default value is dependent on the user's system

File: docs/actions/adb_devices.md (hunk 1 above)
Comment (revolter) [reply docs/actions/adb_devices.md]: It didn't work for some reason, and double spaces are way fragile than a `<br/>`, and can easily be removed by a trailing space removal like I have in Sublime Text. And it's also visible, and so clearer, than some invisible characters.

**Comment by revolter** [comment]
Fixing the review comments in a PR on _fastlane_ before updating the docs again.

**Pull request closed without merging**